src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -84,7 +85,22 @@ public void notifyClaimant(
     ) {
         Map<String, String> parameters = aggregateParams(claim, claim.getResponse()
             .orElseThrow(IllegalStateException::new));
-        notify(claim.getSubmitterEmail(), getEmailTemplates().getClaimantResponseIssued(), parameters, reference);
+
+        String emailTemplate = getClaimantEmailTemplate(claim);
+
+        notify(claim.getSubmitterEmail(), emailTemplate, parameters, reference);
+    }
+
+    private String getClaimantEmailTemplate(Claim claim) {
+        if (claim.getResponse().isPresent()) {
+            Response response = claim.getResponse().get();
```

Comment:
please don't use `.get()` on optional if it can be avoided, use .orElseThrow here as you're in a response class it is guaranteed you'll have one